<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>411</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>581</width>
     <height>391</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout" stretch="3,0,1">
    <item>
     <layout class="QGridLayout" name="gridLayout" columnstretch="2,3,2,3">
      <item row="0" column="2">
       <widget class="QLabel" name="labelFirstName">
        <property name="text">
         <string>First Name</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="labelLastName">
        <property name="text">
         <string>Last Name</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="editLastName"/>
      </item>
      <item row="0" column="3">
       <widget class="QLineEdit" name="editFirstName"/>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="editEmail"/>
      </item>
      <item row="1" column="3">
       <widget class="QLineEdit" name="editPhone"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="labelEmail">
        <property name="text">
         <string>E-mail</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="labelBirthdate">
        <property name="text">
         <string>Birthdate</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="labelPhone">
        <property name="text">
         <string>Phone</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QLabel" name="labelDiscount">
        <property name="text">
         <string>Discount</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QDateEdit" name="dateBirth">
        <property name="displayFormat">
         <string>dd.MM.yyyy</string>
        </property>
        <property name="calendarPopup">
         <bool>true</bool>
        </property>
        <property name="date">
         <date>
          <year>1990</year>
          <month>1</month>
          <day>1</day>
         </date>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QComboBox" name="comboDiscount"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTableView" name="tableHistory">
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="2,3,1,2,2">
      <item>
       <widget class="QPushButton" name="buttonAssign">
        <property name="text">
         <string>Assign Card</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="buttonRFID">
        <property name="text">
         <string>RFID</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="buttonApply">
        <property name="text">
         <string>Apply</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="buttonClose">
        <property name="text">
         <string>Close</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
